docs/getting-started/release-notes/v2_2_0.md

```diff
@@ -41,9 +45,27 @@ Start here:
 Start here:
 ### Zowe CLI
 
+Zowe CLI and related components contain the following bug fixes.
+
 #### Zowe CLI (Core)
 
-Start here:
+- Updated Imperative to address `ProfileInfo` related issues.
+- Fixed the Zowe Daemon binary exiting with an error if the daemon server does not start within 3 seconds.
+-  Updated Imperative to fix CLI commands failing with error "Cannot find module 'ansi-colors'".
+- Alter the zowe daemon disable command to only kill the daemon running for the current user.
 #### Zowe CLI Imperative Framework
 
-Start here:
\ No newline at end of file
+- Expose the isSecured functionality from the ProfilesCredentials. ([#549](https://github.com/zowe/imperative/issues/549))
+- Allow the ConfigAutoStore to store plain-text properties that are defined as secure in the schema (e.g. user, password). ([zowe/vscode-extension-for-zowe: #1804](https://github.com/zowe/vscode-extension-for-zowe/issues/1804))
+- Added ANSI escape codes trimming for the Web Help. ([#704](https://github.com/zowe/imperative/issues/704))
+- Fixed `AbstractRestClient` not converting LF line endings to CRLF for every line when downloading large files on Windows. ([zowe/zowe-cli/#1458](https://github.com/zowe/zowe-cli/issues/1458))
+- Fixed `zowe --version --rfj` including a trailing newline in the version field. ([#842](https://github.com/zowe/imperative/issues/842))
+- Fixed `--response-format-json` option not supported by some commands in daemon mode. ([#843](https://github.com/zowe/imperative/issues/843))
+- Fixed DefaultHelpGenerator unable to find module "ansi-colors" when Imperative is imported.
```

Comment:
This line can be removed for changes between Zowe 2.1 and 2.2, since Zowe 2.1 bundled Imperative 5.3.2 and the "Cannot find module 'ansi-colors'" bug only applied to v5.3.4.